tests/Feature/Modules/Crm/TitleCrudTest.php

```
@@ -0,0 +1,117 @@
+            ->test(ManageTitles::class)
+            ->mountAction(TestAction::make('edit')->table($title), $payload)
+            ->fillForm($payload)
+            ->callMountedAction();
+
+        /* Assert */
+        $this->assertDatabaseHas('partners_titles', [
+            'id'         => $title->id,
+            'name'       => $payload['name'],
+            'short_name' => $payload['short_name'],
+        ]);
+    }
```

PHPMD (2.15.0)

The method it_updates_a_title is not named in camelCase. (undefined)

(CamelCaseMethodName)